client/gutenberg/editor/components/sidebar/last-revision/index.js

```diff
@@ -1,15 +1,26 @@
+/** @format */
 /**
  * External dependencies
  */
 import React from 'react';
 
+/**
+ * Internal dependencies
+ */
+import PostLastRevision from 'gutenberg/editor/components/post-last-revision';
```

Comment:
So we override a default behavior from the original sidebar. I'm wondering whether we should make it filterable for consistency with other parts of the sidebar like `PostFeaturedImage` or `PostTaxonomyType`.

Reply:
Sounds reasonable. I think as long as we have a hookpoint for the link url/click handler on the button, that would accommodate most folks